ui/page/governance/governance_page.go

```diff
@@ -98,68 +84,58 @@ func (pg *Page) HandleUserInteractions() {
 	}
 
 	for pg.enableGovernanceBtn.Clicked() {
-		go pg.consensusPage.FetchAgendas()
 		go pg.WL.MultiWallet.Politeia.Sync()
-		pg.proposalsPage.isSyncing = pg.multiWallet.Politeia.IsSyncing()
+		pg.Display(NewProposalsPage(pg.Load))
 		pg.WL.MultiWallet.SaveUserConfigValue(load.FetchProposalConfigKey, true)
 	}
 
-	if clicked, selectedItem := pg.tabCategoryList.ItemClicked(); clicked {
-		if pg.selectedCategoryIndex != selectedItem {
-			pg.selectedCategoryIndex = selectedItem
-			pg.changed = true
-		}
-
-		// call selected page OnNavigatedTo() only once
-		if pg.changed && pg.selectedCategoryIndex == 0 {
-			pg.proposalsPage.OnNavigatedTo()
-		} else if pg.changed && pg.selectedCategoryIndex == 1 {
-			pg.consensusPage.OnNavigatedTo()
+	if tabItemClicked, clickedTabIndex := pg.tabCategoryList.ItemClicked(); tabItemClicked {
+		if clickedTabIndex == 0 {
+			pg.Display(NewProposalsPage(pg.Load)) // Display should do nothing if the page is already displayed.
+		} else if clickedTabIndex == 1 {
+			pg.Display(NewConsensusPage(pg.Load))
```

Comment:
#980 ensures that attempting to display a page that is already the current page does nothing. Currently, on master, the current page will be closed and the new instance displayed.